.travis.yml

```diff
@@ -7,6 +7,7 @@ services:
 install:
   - pip install pipenv
   - pipenv sync
+  - sudo sysctl -w vm.max_map_count=262144
```

Comment:
are you sure you can run a sudo command here?

Reply:
It seems it's ok :)
https://travis-ci.org/github/idealista/elasticsearch_role/builds/701230398#L198